torch/onnx/_internal/exporter.py

```diff
@@ -635,6 +635,10 @@ def pre_export_passes(
     # Remove them since ONNX inference does not need them.
     module = passes.RemoveInputMutation(diagnostic_context, module).run(*fx_module_args)
 
+    # ONNX does not support concept of (implicit) type promotion.
+    # Insert type casts explicitly where needed.
+    module = passes.ExplicitTypePromotionPass(diagnostic_context, module).run()
```

Comment:
Is there a convention for naming the passes? Looks like the surrounding ones start with verbs

Reply:
good catch, I will name it as `InsertExplicitTypePromotion`.

Reply:
Or would you prefer `InsertTypePromotion`?